### DISCUSSION TITLE
Using newer FolderBrowserDialog

### DISCUSSION THREAD

**Discussion by dmwyatt** [pythonnet/pythonnet#1721] [Q&A] [answered]
I'm an experienced python dev but have close to zero .NET or any Windows-specific experience so I'm just trying to orient myself here.

In my python application thats going to run on Windows I want to use the [newer `FolderBrowserDialog`](https://docs.microsoft.com/en-us/dotnet/core/compatibility/winforms#modernization-of-the-folderbrowserdialog) that I think(?) is only available in .NET Core 3.0 and up.

Via my reading around over the past couple hours it looks like I need to wait for pythonnet 3...correct?

**Comment by lostmsu** [comment, marked as answer]
Correct if you need a stable library for your project.
